.github/workflows/deploy.yml

```diff
@@ -0,0 +1,21 @@
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Dependencies
+        run: npm install -g firebase-tools
+      - name: Checkout Repo
+        uses: actions/checkout@master
+      - name: Install Dependencies
+        run: npm install
```

Comment:
I suppose I should make this clear in the contributing instructions, and remove package-lock.json, but prefer yarn for deps :)

Reply:
Works - https://github.com/attrib/Lostark-helper/actions/runs/3130437161/jobs/5080683736